changelog/dip1000_deprecation_warnings.dd

```diff
@@ -0,0 +1,6 @@
+Print warning messages unless -revert=dip1000 is used
```

Comment:
I guess the little detail of adding this option was overlooked? :-)

Reply:
Yes, unfortunately: https://github.com/dlang/dmd/pull/14019